sass/components/_primary-header.scss

```diff
@@ -0,0 +1,24 @@
+
+/*---main header css---*/
+
+.header {
+  font-size: 2em;
+  font-family: Helvetica, Arial, "Lucida Grande", sans-serif;
+  background-image: url("../imgs/top-bkgrnd.jpg");
+}
+
+header {
```

Comment:
As noted previously, avoid unscoped element selectors inside components. This rule will apply to all `<header>`s